src/site/content/en/metrics/ttfb/index.md

```diff
@@ -41,7 +42,28 @@ Reducing latency in connection setup time and on the backend will contribute to
 
 ### What is a good TTFB score?
 
-Due to the wide variation of network and application backend stacks, an arbitrary number can't be placed on what consists of a "good" TTFB score. Because TTFB precedes [user-centric metrics](/user-centric-performance-metrics/) such as [First Contentful Paint (FCP)](/fcp/) and [Largest Contentful Paint (LCP)](/lcp/), it's recommended that your server responds to navigation requests quickly enough so that the **75th percentile** of users experience an [FCP within the "good" threshold](/fcp/#what-is-a-good-fcp-score).
+Due to the wide variation of network and application backend stacks, an arbitrary number can't be placed on what consists of a "good" TTFB score. Because TTFB precedes [user-centric metrics](/user-centric-performance-metrics/) such as [First Contentful Paint (FCP)](/fcp/) and [Largest Contentful Paint (LCP)](/lcp/), it's recommended that your server responds to navigation requests quickly enough so that the **75th percentile** of users experience an [FCP within the "good" threshold](/fcp/#what-is-a-good-fcp-score). However, as a rough guide, most sites should strive to have Time To First Byte of **0.8 seconds** or less.
```

Comment:
Yeah I struggled with this but think it's still valid that any thresholds we give here are somewhat arbitrary, because they are not [user-centric](https://web.dev/user-centric-performance-metrics/) nor based on a [rigorous, well-defined, process](https://web.dev/defining-core-web-vitals-thresholds/).

What about removing the "can't be placed" wording to somewhat avoid the "we can't recommend, but here's one anyway"?

```suggestion
Due to the wide variation of network and application backend stacks, a "good" TTFB score—and the impact of that—can vary. Because TTFB precedes [user-centric metrics](/user-centric-performance-metrics/) such as [First Contentful Paint (FCP)](/fcp/) and [Largest Contentful Paint (LCP)](/lcp/), it's recommended that your server responds to navigation requests quickly enough so that the **75th percentile** of users experience an [FCP within the "good" threshold](/fcp/#what-is-a-good-fcp-score). However, as a rough guide, most sites should strive to have Time To First Byte of **0.8 seconds** or less.
```

Reply:
> because they are not user-centric nor based on a rigorous, well-defined, process

I think we can remove the entire first sentence and lead with that.

Reply:
LGTM